apps/roboshield/next.config.mjs

```diff
@@ -0,0 +1,29 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
Looking at codeforafrica/charterafrica again, don't we need [`outputFileTracingRoot`](https://nextjs.org/docs/pages/api-reference/next-config-js/output#caveats)?